src/guiapi/include/IWindowMenuItem.hpp

```diff
@@ -39,8 +39,7 @@ class IWindowMenuItem {
     void Enable() { enabled = true; }
     void Disable() { enabled = false; }
     bool IsEnabled() const { return enabled; }
-    void SetHidden() { hidden = true; }
-    void SetNotHidden() { hidden = false; }
+    void SetHidden(bool _hidden) { hidden = _hidden; }
```

Comment:
beware of undescores in var names and macros, some patterns are reserved in the C/C++ standard

Reply:
please ty to avoid bool parameters in methods, consider
```
    void Hide() { hidden = true; }
    void Show() { hidden = false; }
```
instead